### PR TITLE
Copy a project to a new repo.

### PR DESCRIPTION
**Pull request by myaserkhan**
Transferred microverse trial project app to a new repo.

### CODE REVIEW

File: yasir-khan-microverse-trials-main/index.html [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```
Comment (myaserkhan) [inline yasir-khan-microverse-trials-main/index.html:255]: In order to maintain your code clean, you might also wish to delete any commented code from your code base.

**Review by myaserkhan — COMMENTED** [review]
Good job. @MYaserKhan
giphy
Your project doesn't seem to have any linters, though. Please do add them as they keep your code clean.

**Review by jasonfelice — CHANGES_REQUESTED** [review]
Hi @MYaserKhan, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Good use of inline comments✔️
The review looks great ✔️

## Required Changes ♻️
- [ ] According to the project requirements, you are required to add a minimum of 2 comments.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: yasir-khan-microverse-trials-main/index.html [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+          >Which Options describes you best</label
+        >
+        <select name="" id="dropdown" class="form-select">
+          <option value="">Select current role</option>
+          <option value="">Student</option>
+          <option value="">Full time job</option>
+          <option value="">Full time leaner</option>
+          <option value="">Prefer not say</option>
+          <option value="">Other</option>
+        </select>
```
Comment (myaserkhan) [inline yasir-khan-microverse-trials-main/index.html:68]: The value attribute specifies the value to be sent to a server when a form is submitted.

The content between the opening <option> and closing </option> tags is what the browsers will display in a drop-down list. However, the value of the value attribute is what will be sent to the server when a form is submitted.

**Review by jasonfelice — APPROVED** [review]
Hi @MYaserKhan, 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<p align="center"></p>

## Highlights
Used inline comments✔️
The review looks great ✔️
Professional Communication✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**